app/assets/javascripts/student_profile/provided_by_educator_dropdown.js

```diff
@@ -58,12 +61,13 @@
           self.props.onUserDropdownSelect(ui.item.value);
         }
       });
+
+      $('body').bind('click.closeMenu', this.closeMenu);
```

Comment:
Similarly, this affects the whole page which is okay but not ideal.  One option might be to scope the "close to click" to just this component, or another might be too keep the event on `body` like this, but change the event name to be something more unique and specific to this autocomplete widget, so there's no possibility of collision.

This is probably not a real concern, but in general it'd be great to keep components as componentized as we can. :)